packages/angular/src/utils/versions.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { angularDevkitVersion } from './versions';
+
+describe('Angular Versions', () => {
+  it('angular CLI package should use ~ package matching', () => {
+    expect(angularDevkitVersion[0]).toEqual('~');
```

Comment:
Should we perhaps change also the angular version from `^12.1.0` to `~12.1.0` and add it to the check?

Reply:
@FrozenPandaz Seemed to think that we should be able to keep Angular's version at `^12.1.0` as there would be no breaking changes in the minor updates, however, historically, there's been breaking changes with minor updates of angular-cli.